frontend/src/components/ApprovalRequests.vue

```diff
@@ -1,19 +1,17 @@
 <template lang="pug">
-  div
-  v-container#dropdown-example
+  v-container
     v-row
-      v-col(cols='12' sm='3')
-        v-overflow-btn.my-2(:items='dropdown_item' label='Course Name' target='#dropdown-example')
-  v-data-table.elevation-1(v-model='selected' :headers='headers' :items='ta' :single-select='singleSelect' item-key='name' show-select='')
-  div
-    | By clicking Approve,you are certifying that the above student(s) have succesfully performed the ta duty equivaent to * hours per week.
-  .text-center
-    v-row
-      v-col(cols='12' sm='2')
-        v-btn Approve
-      v-col(cols='12' sm='2')
-        v-btn Not Approved
-
+      v-col(cols='12', sm='3')
+        v-overflow-btn.my-2(:items='dropdown_item', label='Course Name', target='#dropdown-example')
```

Comment:
Do you need `target='#dropdown-example'` now?

Reply:
removed unwanted toggle-example